napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -91,28 +90,34 @@ def __init__(self, layer):
         self.depictionComboBox.activated[str].connect(self.changeDepiction)
         self.depictionLabel = QLabel(trans._('depiction:'))
 
-        self.planeControls = QtPlaneControls()
-        self.planeControls.planeThicknessSlider.setValue(
-            self.layer.plane.thickness
-        )
-        self.planeControls.planeThicknessSlider.valueChanged.connect(
-            self.changePlaneThickness
-        )
+        # plane controls
```

Comment:
Could you explain why piecing out these controls is necessary for this PR? The encapsulation of these controls in a single widget was specifically requested when they were added, maintainability being the cited reason
(No link, on mobile but PR was interactive plane controls)